fitbenchmarking/templates/table_template.html

```diff
@@ -68,6 +68,24 @@ <h3>Errors</h3>
                         </tr>
                     {% endfor %}
                 </table>
+		{% if has_pp %}
+		<hr>
+		<section class="accordion">
+                    <input type="checkbox" name="collapse" id="handle3" checked="checked">
+                    <h2 class="handle">
+                    <label for="handle3">Performance profile</label>
+                    </h2>
+                    <div class="content">
+		      <img alt={{ pp_location }} src={{ pp_location }} />
```

Comment:
This should be guarded for any paths with spaces (I think some of mantids minimisers)
```suggestion
		      <img alt="{{ pp_location }}" src="{{ pp_location }}" />
```